lib/gui/lvgl/Kconfig.themes

```diff
@@ -1,56 +1,666 @@
 # Copyright (c) 2018 Jan Van Winkel <jan.van_winkel@dxplore.eu>
+# Copyright (c) 2020 Teslabs Engineering S.L.
 # SPDX-License-Identifier: Apache-2.0
 
 menu "Themes"
 
-config LVGL_THEMES
-	bool "Enable build-in themes"
-	help
-	  Enable build-in themes
+choice
+	prompt "Theme"
+	default LVGL_THEME_MATERIAL
```

Comment:
Better use `LVGL_THEME_EMPTY` as default as it reduces the rom footprint with 3.7k.
Tested the sample on the `nrf` board and it is still OK

Reply:
Empty theme doesn't look really good unless you apply some custom style (it's actually the purpose). So if `EMPTY` is the default I think sample should be adjusted. Note that there is also a gain by disabling logging (now enabled by default).

![image](https://user-images.githubusercontent.com/25011557/88227392-d711f000-cc6d-11ea-8697-775387138b3e.png)

Reply:
I have no problem if `CONFIG_LVGL_THEME_MATERIAL` is added to the sample `prj.conf` file. But I like the idea of setting the defaults in such a way that you get a minimal build and that as a user you need to enable what you need to get your app working.

Reply:
Ok, makes sense. Default is now empty but sample enables material.